src/pydex_app/orderbook.py

```diff
@@ -3,17 +3,28 @@
 
 author: officialcryptomaster@gmail.com
 """
-
+from pydex_app.database import PYDEX_DB as db
 from pydex_app.db_models import SignedOrder
+from pydex_app.order_watcher_client import OrderWatcherClient as owc
 from pydex_app.utils import paginate
 
 
 class Orderbook:
     """Abstraction for orderbook of signed orders
     """
+    _owc = owc()
```

Comment:
OrderWatcher client needs to run in a separate process as a daemon otherwise it will miss updates from the server.

Reply:
Actually, I realized you are only using it here to register the order with the server so it is fine.